services/QuillLMS/app/workers/assign_recommendations_worker.rb

```diff
@@ -4,53 +4,68 @@ class AssignRecommendationsWorker
   include Sidekiq::Worker
   sidekiq_options queue: SidekiqQueue::CRITICAL
 
-  # rubocop:disable Metrics/CyclomaticComplexity
-  def perform(options={})
-    options = options.with_indifferent_access
-    unit_template_id = options["unit_template_id"]
-    classroom_id = options["classroom_id"]
-    student_ids = options["student_ids"]
-    last = options["last"]
-    lesson = options["lesson"]
-    assign_on_join = options["assign_on_join"] || false
-    assigning_all_recommended_packs = options["assigning_all_recommended_packs"]
+  # rubocop:disable Metrics/ParameterLists
+  def perform(
```

Comment:
I like moving to named params with default values over the weird `options` shuffling of the original.

Reply:
FYI, @anathomical I just realized that sidekiq doesn't [support](https://github.com/mperham/sidekiq/issues/2372).

Reply:
Oh, yeah.  I should have remembered that because I got into trouble trying to do something similar a while ago.  Whoops!